### PR TITLE
Debian 12 Install Improvements

### PR DESCRIPTION
**Pull request by jxmx**
- Improve standard Debian 12 installer page
- Add information on UEFI/SecureBoot
- Add note on Cloud Kernel issues with VPS-hosted nodes

### CODE REVIEW

File: docs/adv-topics/uefi-secureboot.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+a cloud virtual machine / VPS, make sure that a remote virtual
+console is connected. Then, reboot the system.
+
+    !!! warning "MOK Load Staging is One-Time Only"
+        Note that the MOK is only staged for load one time. If the MOK load prompt
+        is not responded to at the first/next reboot, the MOK staged for load
+        is wiped and Step #1 will have to be run again.
+
+3. Complete the steps to review, unlock, and load the MOK
+    !!! tip "The next step varies by vendor"
```
Comment (Allan-N) [inline docs/adv-topics/uefi-secureboot.md:79]: Did you want this "tip" to be inline in the doc?  or somehow stand out?

File: docs/user-guide/install.md [hunk 2]
```diff
@@ -1,9 +1,46 @@
 # Debian 12 Install
 
-These instructions are for installing ASL3 on various x86 systems. AllStarLink v3 is supported on Debian 12 Bookworm systems, and those based on Bookworm (e.g. Raspberry Pi OS). Both the x86_64/amd64 and arm64/aarch64 platforms are supported through apt/deb installation packages.
-
-Note that currently the project does not support armv7l/armhf platforms because all known use of AllStarLink is on hardware which supports the Bookworm arm64 distribution such as Raspberry Pi 3, 4, 5, and Zero 2 W. If you have a platform that must use armv7l/armhf 32-bit packages only please file an issue at [ASL3 on GitHub](https://github.com/AllStarLink/ASL3/issues).
-
+These instructions are for installing ASL3 on general purpose operating systems manually. For
+installation in Raspberry Pis, consider using the [AllStarLink 3 Pi Appliance](pi-detailed.md)
```
Comment (Allan-N) [inline docs/user-guide/install.md:4]: Should this be "installation on Raspberry Pis"?

File: docs/user-guide/install.md [hunk 3]
```diff
@@ -1,9 +1,46 @@
...
+
+!!! note "Architecture Support"
+    Currently the project does not support armv7l/armhf platforms because all known use of AllStarLink is on hardware which supports the Bookworm arm64 distribution such as Raspberry Pi 3, 4, 5, and Zero 2 W. If you have a platform that must use armv7l/armhf 32-bit packages only please file an issue at [ASL3 on GitHub](https://github.com/AllStarLink/ASL3/issues).
```
Comment (Allan-N) [inline docs/user-guide/install.md:7]: 1. Currently, (add the comma)
2. all known uses (add an "s")
3. "must use armv7l/armhf 32-bit packages only" --> "can only use armv7l/armhf 32-bit packages" ?

File: docs/user-guide/install.md [hunk 4]
```diff
@@ -57,7 +94,11 @@ from this method of node lookup instead.
 Install with `sudo apt install asl3-update-nodelist`.
 
 ## Node Configuration
-Next step is to configure the node settings. YouTuber Freddie Mac has a nice ASL3 RPi installation and configuration video. See the part where the asl-menu is shown [https://youtu.be/aeuj-yI8qrU](https://youtu.be/aeuj-yI8qrU). Also see [ASL3 Menu](menu.md) for details.
+Next step is to configure the node settings. It is recommended to use the
+`asl-menu` command to set configuration for common use cases. See
```
Comment (Allan-N) [inline docs/user-guide/install.md:98]: "to set the configuration" (add "the")

and a possible alternative to "set" might be "manage"/

**Review by Allan-N — CHANGES_REQUESTED** [review]

File: docs/adv-topics/uefi-secureboot.md (hunk 1 above)
Comment (jxmx) [reply docs/adv-topics/uefi-secureboot.md:79]: Oops. Needed an extra linebreak for that to render right.

File: docs/user-guide/install.md (hunk 3 above)
Comment (jxmx) [reply docs/user-guide/install.md:7]: Fixed

File: docs/user-guide/install.md (hunk 2 above)
Comment (jxmx) [reply docs/user-guide/install.md:4]: Fixed

File: docs/user-guide/install.md (hunk 4 above)
Comment (jxmx) [reply docs/user-guide/install.md]: fixed

**Review by Allan-N — APPROVED** [review]

**Pull request merged**